ui/page/settings/view.jsx

```diff
@@ -481,43 +480,47 @@ class SettingsPage extends React.PureComponent<Props, State> {
             />
 
             {(isAuthenticated || !IS_WEB) && (
-              <Card
-                title={__('Blocked Channels')}
-                actions={
-                  <p>
-                    <React.Fragment>
-                      {userBlockedChannelsCount === 0
-                        ? __("You don't have blocked channels.")
-                        : userBlockedChannelsCount === 1
-                        ? __('You have one blocked channel.') + ' '
-                        : __('You have %channels% blocked channels.', { channels: userBlockedChannelsCount }) + ' '}
-                      {
-                        <Button
-                          button="link"
-                          label={userBlockedChannelsCount === 0 ? null : __('Manage')}
-                          navigate={`/$/${PAGES.BLOCKED}`}
-                        />
-                      }
-                    </React.Fragment>
-                  </p>
-                }
-              />
+              <>
+                <Card
+                  title={__('Notifications')}
```

Comment:
maybe we could introduce tabs to settings instead of this? (not blocking feedback, can evolve)

Reply:
Yeah. This definitely a step towards some tabbed/nested setting page.